packages/gatsby-cli/src/__tests__/handlers/__snapshots__/plugin-add.ts.snap

```diff
@@ -0,0 +1,104 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
For anyone reading this (or my future self), I took a look at making the quote type of the output strings in the transform consistent (e.g. all backticks).

The only obstacle is in the `buildPluginNode` function we `JSON.stringify` the options object, and if we try to make object keys backticks then the transform fails since that's invalid JSON (and we can't expect keys to only be alphabetic).

Anyhow not a huge deal, but wanted to leave this for posterity. Will leave this out of this PR.